client/src/modules/stock/requisition/modals/action.modal.html

```diff
@@ -69,7 +68,7 @@
     <button type="button" class="btn btn-default" ng-click="$ctrl.cancel()" data-method="cancel" translate>
       FORM.BUTTONS.CLOSE
     </button>
-    <bh-loading-button loading-state="ModalForm.$loading" disabled="ModalForm.$loading">
+    <bh-loading-button loading-state="$ctrl.loading" disabled="$ctrl.loading">
```

Comment:
This would actually work, if you used `bh-submit` instead of `ng-submit` at the top of the form.  That way you wouldn't have to manage this variable yourself in the controller.  Would you prefer doing it that way?